pkg/manager/member/tikv_member_manager.go

```diff
@@ -26,14 +26,14 @@ import (
 	"github.com/pingcap/tidb-operator/pkg/manager"
 	"github.com/pingcap/tidb-operator/pkg/pdapi"
 	"github.com/pingcap/tidb-operator/pkg/util"
-	apps "k8s.io/api/apps/v1beta1"
+	apps "k8s.io/api/apps/v1"
```

Comment:
upgrade from `v1beta1` to `v1` here can cause the pods to recreate or not? do you have a test manually?

Reply:
oh, it's might be a problem to upgrade an existing operator in the cluster.

I think it will fail here because the API version does not match. I guess we need to do API objects migration at the startup of our new operator.

Reply:
made a mistake, in a cluster which supports both v1beta1 & v1 of stateful sets, kube-apiserver will do conversion between them, the client can CRUD different versions of an object with the same name.

Anyway, we should verify it if we can update the operator gracefully from the old operator.

Reply:
> upgrade from `v1beta1` to `v1` here can cause the pods(pd/tikv/tidb) to recreate or not? do you have a test manually?

In stability test, `UpgradeOperatorOrDie` will ensure whether the TiDB cluster will be recreated or not.
I also test it mannuly,and the upgrading won't cause TiDB cluster recreated.